### ISSUE TITLE
Failing test in `tests/listblock/listblock` on IE 11

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#5498]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)
<img width="628" alt="Failed test: The aria-selected attribute of the selected item has the correct value. Expected true, got null." src="https://github.com/ckeditor/ckeditor4/assets/1078728/12a4e492-a878-4560-99f1-c17959b5f46f">

## Other details

*   Browser: IE11
*   OS: Windows 7
*   CKEditor version: `master`
*   Installed CKEditor plugins: `listblock`
